PromisesObjC.podspec

```diff
@@ -28,10 +28,12 @@ Pod::Spec.new do |s|
   }
 
   s.test_spec 'Tests' do |ts|
+    ts.platforms = {:ios => '8.0', :osx => '10.10', :tvos => '9.0'}
```

Comment:
Could you leave a comment noting that the goal of this specification is to omit watchOS, since XCTest is not available to watchOS?

Reply:
Sure. I added a comment.

Reply:
Thanks!